Makefile

```diff
@@ -32,6 +32,11 @@ dev staging: assert-dom0 ## Configures and builds a dev or staging environment
 build-rpm: ## Build RPM package
 	USE_BUILD_CONTAINER=true $(CONTAINER) ./scripts/build-rpm.sh
 
+# Dev RPMs don't contain power settings adjustment (lid)
```

Comment:
IMO the build process should produce env-neutral packages, with env-specific stuff being set up at install- or run-time.